### PR TITLE
add PR and issue template

### PR DESCRIPTION
**Pull request by vuvoth**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md
Comment (Meyanis95) [inline .github/ISSUE_TEMPLATE.md:1]: Can I suggest this template?

**What is the bug?** A concise, high level description of the bug and how it affects you

**How can it be reproduced? (optional)** Include steps, code samples, replits, screenshots and anything else that would be helpful to reproduce the problem.

**Additional context (optional)** Add any other context about the problem here.

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (Meyanis95) [inline .github/PULL_REQUEST_TEMPLATE.md:1]: WDYT of this one?

## Motivation

Describe why this issue should be fixed and link to any relevant design docs, issues or other relevant items.

## Change Summary

Describe the changes being made in 1-2 concise sentences.

## Merge Checklist

_Choose all relevant options below by adding an `x` now or at any time before submitting for review_

- [ ] PR title adheres to the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) standard
- [ ] PR has a [changeset](https://github.com/privacy-scaling-explorations/anon-aadhaar/CHANGELOG.md)
- [ ] PR has been tagged with a change label(s) (i.e. documentation, feature, bugfix, or chore)
- [ ] PR includes documentation if necessary.

## Additional Context

If this is a relatively large or complex change, provide more details here that will help reviewers

Comment (vuvoth) [reply .github/PULL_REQUEST_TEMPLATE.md:1]: Oh! It look nice! I will change my PR!

**Review by Meyanis95 — APPROVED** [review]
LFG!

**Pull request merged**